src/ol/interaction/translateinteraction.js

```diff
@@ -0,0 +1,126 @@
+goog.provide('ol.interaction.Translate');
+
+goog.require('ol.interaction.Pointer');
+
+
+
+/**
+ * @classdesc
+ * Interaction for translating (moving) features.
+ *
+ * @constructor
+ * @extends {ol.interaction.Pointer}
+ * @param {olx.interaction.TranslateOptions} options Options.
+ * @api
+ */
+ol.interaction.Translate = function(options)
+    {
+  goog.base(this, {
+    handleDownEvent: ol.interaction.Translate.handleDownEvent_,
+    handleDragEvent: ol.interaction.Translate.handleDragEvent_,
+    handleUpEvent: ol.interaction.Translate.handleUpEvent_
+  });
+
+  /**
+    * The last position we translated to.
+    * @type {ol.Coordinate}
+    * @private
+    */
+  this.lastCoordinate_ = null;
+
+
+  /**
+   * @type {ol.Collection.<ol.Feature>}
+   * @private
+   */
+  this.features_ = options.features;
+};
+goog.inherits(ol.interaction.Translate, ol.interaction.Pointer);
+
+
+/**
+ * @inheritDoc
+ */
+ol.interaction.Translate.prototype.setMap = function(map) {
+  goog.base(this, 'setMap', map);
+};
+
```

Comment:
Ah, correct. I had been storing the map until I realized I can get it from the event, if I need it.